testflinger/__init__.py

```diff
@@ -64,8 +63,8 @@ def create_flask_app():
     # Otherwise load it from testflinger.conf in the testflinger dir
     config_file = os.environ.get("TESTFLINGER_CONFIG", "testflinger.conf")
     tf_app.config.from_pyfile(config_file, silent=True)
-    if not os.path.exists(tf_app.config["DATA_PATH"]):
-        os.makedirs(tf_app.config["DATA_PATH"])
+    if not tf_app.config["DATA_PATH"].exists():
+        Path.mkdir(tf_app.config["DATA_PATH"])
```

Comment:
let's go ahead and remove the conditional and just do Path.mkdir(tf_app.config["DATA_PATH"], exist_ok=True) since I don't think we need to worry about (< python3.5) at this point